Updated Minimum Requirements for Kaptain Installation. (#4401)
mesosphere/dcos-docs-site: pages/dkp/kaptain/2.0.0/install/dkp/index.md

## Requirements

Before proceeding, verify that your cluster has freely available the following resources:

- 4 CPU cores
- 16 GiB of RAM

> - 170 GiB or Storage Memory

Reviewer: is this 
```suggestion
- 170 GiB of Storage
```
or 
```suggestion
- 170 GiB of Memory
```
?

https://v0-6.kubeflow.org/docs/started/k8s/overview/

## Requirements

> Before proceeding, verify that your cluster has freely available the following resources:

Reviewer 2: ```suggestion
Before proceeding, verify that your cluster has the following resources freely available:
```

- 3 worker nodes

> - 4 CPU cores

Reviewer 3: so these are not requirements per node?

Reviewer 3: if there are still per-node reqs, let's update them but keep the format of the previous

> Before proceeding, verify that your cluster has the following resources freely available:
> 
> - 3 worker nodes
> - 4 CPU cores
> - 170 GiB of Storage
> - 16 GiB of RAM
> - NVIDIA GPU instances are supported only

Reviewer 4: ```suggestion
Before proceeding, verify that your cluster has the following resources freely available:

- Cluster consists of at least 3 worker nodes
- 25 CPU cores
- 32 GiB of RAM
- 170 GiB of Storage
- NVIDIA GPU instances are supported only
```

Reviewer 5: committing this so we can kick off the PR preview.
